### PR TITLE
update to rails 5.1.3 and modify active record validations for rails 5

### PR DESCRIPTION
**Pull request by Salanoid**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -11,13 +11,11 @@ class User
...
-  validates_presence_of :email
-  validates_presence_of :fname
-  validates_presence_of :password, on: :create
-  validates_presence_of :password_confirmation, on: :create
-  validates_confirmation_of :password, on: :create
+  validates :email, presence: true, uniqueness: true
+  validates :fname
+  validates :password, on: :create
+  validates :password_confirmation, on: :create
   validates :password, confirmation: true, on: :create
```
Comment (nsantiago2719) [inline app/models/user.rb]: It just needs a validation for the 3 lines above. Anyways good job and thanks.

Comment (Salanoid) [inline app/models/user.rb]: Yeah but in rails 5 there is a new way of validations http://guides.rubyonrails.org/active_record_validations.html . It dosen't work?

Comment (Salanoid) [inline app/models/user.rb]: And Rails 5.1.3 is the last stable release.

Comment (nsantiago2719) [inline app/models/user.rb]: Hi, you need to specify a validation for that. on line 15 in just says `validates :fname` and ActiveRecord doesn't know how to validate it. You can also check the travis ci build to see what's wrong on the code.

**Comment by Salanoid** [comment]
Now it should be ok.

**Comment by nsantiago2719** [comment]
cool. now we can merge this one in master.

**Pull request merged**